test/model/binding.dart

```diff
@@ -521,9 +530,25 @@ class FakeAndroidNotificationHostApi implements AndroidNotificationHostApi {
       groupKey: groupKey,
       inboxStyle: inboxStyle,
       isGroupSummary: isGroupSummary,
+      messagingStyle: messagingStyle,
+      number: number,
       smallIconResourceName: smallIconResourceName,
     ));
+
+    if (tag != null) {
+      _activeNotificationsMessagingStyle[tag] = messagingStyle != null
+        ? MessagingStyle(
+            user: messagingStyle.user,
+            conversationTitle: messagingStyle.conversationTitle,
+            messages: messagingStyle.messages.toList(), // clone to avoid shared-references
```

Comment:
Oh, this should remove icons from the senders. That way it accurately simulates the behavior of our Kotlin code.